boards/pic32-wifire/include/periph_uart.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016, Imagination Technologies Limited and/or its
+ *                 affiliated group companies.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+#ifndef _PERIPH_UART_H_
+#define _PERIPH_UART_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define U4RXR     0xBF801480
+#define RPF8R     0xBF801660
+#define PORTFCLR  0xBF860524
+#define TRISFCLR  0xBF860514
+#define TRISFSET  0xBF860518
+#define ODCFCLR   0xBF860544
```

Comment:
No, this is fixed in my WIP branch (https://github.com/neiljay/RIOT/tree/wip) there is another vendor supplied file with all these definitions in. I'll submit another PR for this, this one is already too big.